Sources/Rendering/Core/ImageCPRMapper/Constants.js

```diff
@@ -0,0 +1,9 @@
+export const MipMode = {
```

Comment:
Average (i.e. AverageIP) is not a Mip (Maximum Intensity Projection).
Therefore I suggest to rename MipMode into BlendMode, SlabMode or ProjectionMode.